app/model/exactTarget/DataExtensionRow.scala

```diff
@@ -163,10 +167,60 @@ object PaperHomeDeliveryWelcome1DataExtensionRow {
   }
 }
 
+object HolidaySuspensionBillingScheduleDataExtensionRow {
+
+  def apply(
+           email: Option[String],
+           saltuation: String,
+           subscriptionName: String,
+           subscriptionCurrency: Currency,
+           packageName: String,
+           billingSchedule: BillingSchedule,
+           numberOfSuspensionsLinedUp: Int,
+           daysUsed: Int,
+           daysAllowed: Int
+         ): HolidaySuspensionBillingScheduleDataExtensionRow = {
+
+    val (thereafterBill, trimmedSchedule) = BillingSchedule.rolledUp(billingSchedule)
+    val count = new AtomicInteger()
+    val futureBills = trimmedSchedule.flatMap(bill => {
```

Comment:
Ok I'll fix that when I rebase https://github.com/guardian/subscriptions-frontend/pull/547 to master and resubmit it.